mergify_engine/actions/dismiss_reviews.py

```diff
@@ -58,9 +58,7 @@ class DismissReviewsAction(actions.Action):
             [types.GitHubLogin],
             FROM_REQUESTED_REVIEWERS,
         ),
-        voluptuous.Required("message", default=None): voluptuous.Any(
-            None, types.Jinja2
-        ),
+        voluptuous.Required("message", default=None): types.Jinja2WithNone,
```

Comment:
unrelated :(

Reply:
I prepared the record with the other change :)